lib/realtime_signs.ex

```diff
@@ -21,11 +21,12 @@ defmodule RealtimeSigns do
         Engine.Departures,
         Engine.Static,
         Engine.Alerts,
+        Engine.BusPredictions,
+        Engine.ChelseaBridge,
```

Comment:
That's a good question. I'm inclined to say that since the bridge is now an integral part of the app, those environment variables should be required. In any case I'll update the README so people know how to populate them.

Reply:
Sounds reasonable to me!